extensions/blocks/dialogue/edit.js

```diff
@@ -0,0 +1,279 @@
+	function hasStyle( style ) {
+		if ( isCustomParticipant || ! participantsFromContext ) {
+			return attributes?.[ style ];
+		}
+
+		return currentParticipant?.[ style ];
+	}
+
+	/**
+	 * Helper to toggle the value of the given style
+	 * It handles local and global (conversation) level.
+	 *
+	 * @param {string} style - style to toggle.
+	 * @returns {void}
+	 */
+	function toggleParticipantStyle( style ) {
+		if ( isCustomParticipant || ! participantsFromContext ) {
+			return setAttributes( { [ style ]: ! attributes[ style ] } );
+		}
+
+		transcritionBridge.updateParticipants( {
+			participantSlug: currentParticipantSlug,
+			[ style ]: ! currentParticipant[ style ],
+		} );
+	}
+
+	/**
+	 * Helper to build the CSS classes for the participant label.
+	 * It handles local and global (conversation) level.
+	 *
+	 * @returns {string} Participant CSS class.
+	 */
+	function getParticipantLabelClass() {
+		if ( isCustomParticipant || ! participantsFromContext ) {
+			return classnames( `${ baseClassName }__participant`, {
+				[ 'has-bold-style' ]: attributes?.hasBoldStyle,
+				[ 'has-italic-style' ]: attributes?.hasItalicStyle,
+				[ 'has-uppercase-style' ]: attributes?.hasUppercaseStyle,
+			} );
+		}
+
+		return classnames( `${ baseClassName }__participant`, {
+			[ 'has-bold-style' ]: currentParticipant?.hasBoldStyle,
+			[ 'has-italic-style' ]: currentParticipant?.hasItalicStyle,
+			[ 'has-uppercase-style' ]: currentParticipant?.hasUppercaseStyle,
+		} );
+	}
+
+	function setShowTimeStamp( value ) {
+		if ( isCustomParticipant || ! participantsFromContext ) {
+			return setAttributes( { showTimeStamp: value } );
+		}
+
+		transcritionBridge.setAttributes( { showTimeStamp: value } );
+	}
+
+	return (
+		<div className={ className }>
+			<BlockControls>
+				{ currentParticipant && isFocusedOnParticipantLabel && (
+					<ToolbarGroup>
+						<ToolbarButton
+							icon="editor-bold"
+							isPressed={ hasStyle( 'hasBoldStyle' ) }
+							onClick={ () => toggleParticipantStyle( 'hasBoldStyle' ) }
+						/>
+
+						<ToolbarButton
+							icon="editor-italic"
+							isPressed={ hasStyle( 'hasItalicStyle' ) }
+							onClick={ () => toggleParticipantStyle( 'hasItalicStyle' ) }
+						/>
+
+						<ToolbarButton
+							icon={ formatUppercase }
+							isPressed={ hasStyle( 'hasUppercaseStyle' ) }
+							onClick={ () => toggleParticipantStyle( 'hasUppercaseStyle' ) }
+						/>
+					</ToolbarGroup>
+				) }
+			</BlockControls>
+
+			<InspectorControls>
+				<Panel>
+					<PanelBody title={ __( 'Participant', 'jetpack' ) }>
+						<ParticipantsControl
+							className={ baseClassName }
+							participants={ participants }
+							participantSlug={ participantSlug || '' }
+							onSelect={ setAttributes }
+						/>
+						<ParticipantControl
+							className={ className }
+							participantValue={ participant }
+							onChange={ setAttributes }
+						/>
+					</PanelBody>
+
+					<PanelBody title={ __( 'Time stamp', 'jetpack' ) }>
```

Comment:
```suggestion
					<PanelBody title={ __( 'Timestamp', 'jetpack' ) }>
```

Reply:
Asked yo Saul about this, in a previous commit suggestion.

Reply:
let's change it